### DISCUSSION TITLE
Load JSON file from local storage.

### DISCUSSION THREAD

**Discussion by carlotamburin** [naver/billboard.js#2507] [Q&A]
Hello, I have a questions. I need to do chart from JSON file on local computer. How I can load that file? If i require it the chart won't load. 

Thank you in advance,

Carlo

**Comment by netil** [comment]
Loading doesn't matter where the data  is coming from.
If data stored on localStorage, it needed to be transformed a valid format that billboard.js accepts.

```js
// if the json data stored as:
localStorage.data = JSON.stringify([{name: "www.site1.com", upload: 800, download: 500, total: 400},{name: "www.site2.com", upload: 600, download: 600, total: 400}]);

bb.generate({
	data: {
		json: localStorage.data,  // just specify data source
		keys: { ... }
	}
});
```
